situp-benchmarks/service-map-stateful-benchmarks/READEME.md

```diff
@@ -0,0 +1,24 @@
+# Service Map Stateful Benchmarks
+
+This package contains benchmarks for the service map stateful processor using JMH: https://openjdk.java.net/projects/code-tools/jmh/ . 
+
+Integration with gradle is done with the following gradle plugin for JMH: https://github.com/melix/jmh-gradle-plugin.
+
+The plugin creates a source set for the JMH benchmarks, and provides a few gradle tasks for running and building the benchmarks.
+
+## Running the tests via gradle task
+
+Tests can be run via the "jmh" gradle task provided by the plugin. The README for the plugin provides the various parameters that
+can be provided to the plugin. 
```

Comment:
Might better also add the example command with args here

Reply:
so the gradle task actually doesn't take arguments. you can only edit those parameters via the actual gradle task setup and overriding them. I did find some guidance that we can map them to project properties, if we need to.

Reply:
I see. I misunderstood that this will take the same args just as below. It will be nice if it does. But I am OK with it.